integration-cli/docker_cli_inspect_test.go

```diff
@@ -79,19 +81,17 @@ func (s *DockerSuite) TestInspectStatus(c *check.C) {
 }
 
 func (s *DockerSuite) TestInspectTypeFlagContainer(c *check.C) {
-	testRequires(c, DaemonIsLinux)
 	//Both the container and image are named busybox. docker inspect will fetch container
 	//JSON State.Running field. If the field is true, it's a container.
 
-	dockerCmd(c, "run", "--name=busybox", "-d", "busybox", "top")
+	dockerCmd(c, "run", "--name=busybox", "-d", "busybox", "sleep", "60")
```

Comment:
Good point. Don't know why I didn't notice that as I updated so many others yesterday to do exactly that... Will be in next push.